UniversalAlgebra.tex

```diff
@@ -67,30 +67,31 @@ \section{Universe, Type, and Signature}
 
 \begin{itemize}
 
-    \item A \textit{signature} is a pair $\Sigma = (S,F)$ such that $S$ is a set
-    of single sort. $F$ is the collection of operation names.
-
-    \item A \emph{$\Sigma$-algebra} $A$ is defined as pair $A = (A,F_A)$, a
-    mathematical structure that has a domain $(A)$ and a family of functions
-    $(F_A)$ defined for each function symbol in the signature on this domain.
-    Algebra provides an interpretation for the carrier set $A$ and function
-    symbols $F_A$ of a theory.
-
-    \item Let $E$ be a set of equations, then model $(Mod_\Sigma(E))$ denotes
-    the class of $\Sigma$-algebra satisfying all the equations in $E$.
-    \[Mod_\Sigma(E) = \{ A | A \text{ is a } \Sigma-algebra \text{ and A } \models E \} =
-    Mod[⟨\Sigma,E⟩]\]
+    \item A \textit{signature} is a pair $\Sigma = (S,F)$ such that $S$ is a
```

Comment:
See, this is still wrong! For single-sorted universal algebra $S$ is a 'sort symbol'. You should look this up instead of making it up.